src/features/island/collectibles/CollectibleCollection.tsx

```diff
@@ -412,6 +414,10 @@ export const COLLECTIBLE_COMPONENTS: Record<
   "Town Sign": Sign,
   "White Crow": WhiteCrow,
   Bud: Bud,
+
+  // Fishing Season SFTs
```

Comment:
Consider: Use official name of the season (what the banner and ticket are called).